diff --git a/Wave_PostProcessingOrchestra.py b/Wave_PostProcessingOrchestra.py
index 6225a5b..7a2734a 100644
--- a/Wave_PostProcessingOrchestra.py
+++ b/Wave_PostProcessingOrchestra.py
@@ -12,10 +12,6 @@ import subprocess
 import sys
 import config
 from colorama import Fore
-import Filelist_Generation
-import GENERIC_exh_postprocessing
-import Collapse_Results_ToSummary
-import Collapse_Results_ToDaily
 
 ################
 ### SWITCHES ###
@@ -25,8 +21,8 @@ import Collapse_Results_ToDaily
 RUN_PAMPRO_MERGE_METAFILES = 'No'           # This needs running if accelerometer files are processed through Pampro to merge metafiles into 1 for each person.
 RUN_PAMPRO_COLLATE_ANOMALIES = 'No'         # This needs running to include information on anomalies if files are processed through Pampro.
 RUN_FILELIST_GENERATION = 'No'              # Create a filelist of all Wave output files. Depending on what is specified in the config.py, this might be all available or only the new files (that are not in the appended summary file)
-RUN_GENERIC_EXH_POSTPROCESSING = 'No'       # Generic exhaustive post processing works through generic updates to the Wave output. This needs to be completed before collapsing any data.
-RUN_COLLAPSE_RESULTS_TO_SUMMARY = 'Yes'      # Takes the file created from exhaustive post processing and collapses it down to a summary (1 line) file. These are saved as individual files.
+RUN_GENERIC_EXH_POSTPROCESSING = 'Yes'       # Generic exhaustive post processing works through generic updates to the Wave output. This needs to be completed before collapsing any data.
+RUN_COLLAPSE_RESULTS_TO_SUMMARY = 'No'      # Takes the file created from exhaustive post processing and collapses it down to a summary (1 line) file. These are saved as individual files.
 RUN_COLLAPSE_RESULTS_TO_DAILY = 'No'        # Takes the file created from exhaustive post processing and collapses it down to a daily (1 line per day) file. These are saved as individual files.
 RUN_APPEND_SUMMARY_FILES = 'No'             # Appends together the individual summary level data files (created within the collapse results to summary file)
 RUN_APPEND_DAILY_FILES = 'No'               # Appends together the daily level data files (created within the collapse results to summary file)
@@ -41,7 +37,7 @@ venv_python = sys.executable
 
 # Running script:
 def run_script(script):
-    filelist_script_path = os.path.join(config.ANALYSIS_FOLDER, script)
+    filelist_script_path = os.path.join(config.ROOT_FOLDER, config.ANALYSIS_FOLDER, script)
     if not os.path.exists(filelist_script_path):
         print(f"Error: The script {filelist_script_path} does not exist.")
         return
@@ -62,46 +58,32 @@ if __name__ == '__main__':
             run_script("Pampro_Collate_Anomalies.py")
 
     # Running the Filelist_Generation script:
-    #!!!! use optional argument num_jobs, and make several filelists
     if RUN_FILELIST_GENERATION.lower() == 'yes':
         print_message("CREATING FILELIST")
-        #run_script("Filelist_Generation.py")
-        Filelist_Generation.create_folders()
-        Filelist_Generation.create_filelist()
-        Filelist_Generation.remove_files()
+        run_script("Filelist_Generation.py")
 
     # Running the GENERIC_EXH_POSTPROCESSING script:
     if RUN_GENERIC_EXH_POSTPROCESSING.lower() == 'yes':
         print_message("COMPLETING GENERIC EXHAUSTIVE ANALYSIS")
-        #run_script("GENERIC_exh_postprocessing.py")
-        GENERIC_exh_postprocessing.main()
-
-    # Running the Collapse_Results_ToSummary script:
-    if RUN_COLLAPSE_RESULTS_TO_SUMMARY.lower() == 'yes':
-        print_message("COLLAPSING DATA TO INDIVIDUAL SUMMARY FILES")
-        #run_script("Collapse_Results_ToSummary.py")
-        Collapse_Results_ToSummary.main()
-
-    # Running the Collapse_Results_ToDaily script:
-    if RUN_COLLAPSE_RESULTS_TO_DAILY.lower() == 'yes':
-        print_message("COLLAPSING DATA TO INDIVIDUAL DAILY FILES")
-        #run_script("Collapse_Results_ToDaily.py")
-        Collapse_Results_ToDaily.main()
-
-    # Running the Append_Summary_Files script:
-    if RUN_APPEND_SUMMARY_FILES.lower() == 'yes':
-        print_message("APPENDING ALL INDIVIDUAL SUMMARY FILES TOGETHER")
-        run_script("Appending_Summary_Files.py")
-
-    # Running the Append_Daily_Files script:
-    if RUN_APPEND_DAILY_FILES.lower() == 'yes':
-        print_message("APPENDING ALL INDIVIDUAL DAILY FILES TOGETHER")
-        run_script("Appending_Daily_Files.py")
-
-    # Running the Append_Hourly_Files script:
-    if RUN_APPEND_HOURLY_FILES.lower() == 'yes':
-        print_message("APPENDING ALL INDIVIDUAL HOURLY FILES TOGETHER")
-        run_script("Appending_Hourly_Files.py")
+        run_script("GENERIC_exh_postprocessing.py")
+
+    # Running the Collapse_Results script (to Summary and/or Daily:
+    if RUN_COLLAPSE_RESULTS_TO_SUMMARY.lower() == 'yes' or RUN_COLLAPSE_RESULTS_TO_DAILY.lower() == 'yes':
+        if RUN_COLLAPSE_RESULTS_TO_SUMMARY.lower() == 'yes':
+            print_message("COLLAPSING DATA TO INDIVIDUAL SUMMARY FILES")
+        if RUN_COLLAPSE_RESULTS_TO_DAILY.lower() == 'yes':
+            print_message("COLLAPSING DATA TO INDIVIDUAL DAILY FILES")
+        run_script("Collapse_Results.py")
+
+    # Running the Append_Files script (Appending summary and/or daily and/or hourly):
+    if RUN_APPEND_SUMMARY_FILES.lower() == 'yes' or RUN_APPEND_DAILY_FILES.lower() == 'yes' or RUN_APPEND_HOURLY_FILES.lower() == 'yes':
+        if RUN_APPEND_SUMMARY_FILES.lower() == 'yes':
+            print_message("APPENDING ALL INDIVIDUAL SUMMARY FILES TOGETHER")
+        if RUN_APPEND_DAILY_FILES.lower() == 'yes':
+            print_message("APPENDING ALL INDIVIDUAL DAILY FILES TOGETHER")
+        if RUN_APPEND_HOURLY_FILES.lower() == 'yes':
+            print_message("APPENDING ALL INDIVIDUAL HOURLY FILES TOGETHER")
+        run_script("Appending_Files.py")
 
     # Running the Verification_Checks script:
     if RUN_VERIFICATION_CHECKS.lower() == 'yes':
